.github/workflows/build.yml

```diff
@@ -50,30 +51,37 @@ jobs:
           const latest34ASan = "${{ steps.latest_commit_3_4_asan.outputs.commit }}"
           const { owner, repo } = context.repo
           let { data: release } = await github.rest.repos.getLatestRelease({ owner, repo })
-          const firstLine = release.body.split('\n')[0]
-          const latestReleaseCommit = firstLine.split('@')[1]
+          const latestReleaseCommit = release.body.split('\n')[0].split('@')[1].trim()
+          // Entry in body may not exist, but if it doesn't the should_build_3_4_asan is still correct
+          const latestRelease34ASanCommit = ((release.body.split('\n')[1] ?? "").split('@')[1] ?? "").trim()
           console.log(`Latest release commit: ${latestReleaseCommit}`)
+          console.log(`Latest 3.4-asan release commit: ${latestRelease34ASanCommit}`)
```

Comment:
Can you move this next to `Latest 3.4-asan:` so it's easier to compare, example from your build: https://github.com/DataDog/ruby-dev-builder/actions/runs/13411682947/job/37463108782#step:5:25

Could even align all 4 commits vertically to make it even easier.